### PR TITLE
Fix broken functional test

### PR DESCRIPTION
**Pull request by ascott1**
This corrects the page title in the functional test scenario.
## Review

@marteki

### CODE REVIEW

**Comment by Scotchester** [comment]
We have GOT to come up with a better method for testing this kind of thing. :confounded:

**Comment by higs4281** [comment]
:+1: tests run for me without failures.

File: test/functional/dd-functional-settlement-spec.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ describe( 'A dynamic financial aid disclosure that\'s required by settlement', f
 
 	// TODO - Change this title when we move from demo mode.
```
Comment (marteki) [inline test/functional/dd-functional-settlement-spec.js]: We can remove this TODO comment now! :smile: :tada:

**Comment by marteki** [comment]
All tests passing for me as well! :+1:

**Comment by ascott1** [comment]
Removed the TODO comment

**Pull request merged**